packages/circuit-ui/components/Popover/Popover.spec.tsx

```diff
@@ -41,43 +42,44 @@ describe('PopoverItem', () => {
     return renderFn(<PopoverItem {...props} />);
   }
 
-  const baseProps = { children: 'PopoverItem' };
+  const baseProps = {
+    children: 'PopoverItem',
+    icon: Download as FC<IconProps>,
+  };
```

Comment:
Why do you need to type the icon here explicitly? 🤔

Reply:
I think this would also work:
```suggestion
  const baseProps = {
    children: 'PopoverItem',
    icon: Download,
  } as const;
```

Reply:
Yes, opening a PR...

Reply:
Here it is: https://github.com/sumup-oss/circuit-ui/pull/1615

It doesn't fix it either though 😕 It seems like the `sizes` union in the `IconProps` has to match exactly.

Reply:
That's okay, I guess 🤷

Did you also see the type errors in these specs when working locally on your other branch? Or was that only my setup?

Reply:
Yes, I saw them too. I think we need to add a `tsconfig.eslint.json` file inside `packages/circuit-ui` in order for the errors to show up in CI.

Reply:
No, all good to go. We can continue to discuss the `IconProps` issue in the other PR.